src/main/java/com/pi4j/driver/display/BaseGraphicsDisplayComponent.java

```diff
@@ -0,0 +1,73 @@
+package com.pi4j.driver.display;
+
+public class BaseGraphicsDisplayComponent {
+
+    protected final GraphicsDisplayDriver driver;
+
+    public BaseGraphicsDisplayComponent(GraphicsDisplayDriver driver) {
+        this.driver = driver;
+    }
+
+    public void fill(int x, int y, int width, int height, java.awt.Color color) throws java.io.IOException {
+
+        byte[] data = new byte[width * height * 16 / 8];
```

Comment:
Optional: I'd just prepare a single horizontal line and then send it `height` times at different y positions